homeassistant/components/notify/smtp.py

```diff
@@ -241,14 +241,16 @@ def _build_html_msg(text, html, images):
     alternative.attach(MIMEText(html, ATTR_HTML, _charset='utf-8'))
     msg.attach(alternative)
 
-    for atch_num, atch_name in enumerate(images):
-        name = os.path.basename(atch_name)
-        try:
-            with open(atch_name, 'rb') as attachment_file:
-                attachment = MIMEImage(attachment_file.read(), filename=name)
-            msg.attach(attachment)
-            attachment.add_header('Content-ID', '<{}>'.format(name))
-        except FileNotFoundError:
-            _LOGGER.warning("Attachment %s [#%s] not found. Skipping",
-                            atch_name, atch_num)
+    if images is not None:
```

Comment:
This should be enough.
```python
if images:
```

Reply:
It is best practice and recommended to use `is not None` (see [Python docs](https://www.python.org/dev/peps/pep-0008/#programming-recommendations))

> Comparisons to singletons like None should always be done with `is` or `is not`, never the equality operators.
>
> Also, beware of writing `if x` when you really mean `if x is not None` -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be `false` in a boolean context!

Reply:
Thanks for pointing that out.